pkg/managementconflict/annotations_test.go

```diff
@@ -56,7 +56,7 @@ func TestGetManagementConflictPreventionAnnotationValue(t *testing.T) {
 		t.Run(tc.Name, func(t *testing.T) {
 			obj := unstructured.Unstructured{}
 			obj.SetAnnotations(tc.Annotations)
-			policy, err := GetManagementConflictPreventionAnnotationValue(&obj)
+			policy, err := GetManagementConflictPreventionPolicy(&obj)
```

Comment:
"ShouldSucceed" values for some test cases may need to be updated?

Reply:
Good call, will fix - thank you!